packages/react/src/components/CodeSnippet/__tests__/CodeSnippet-test.js

```diff
@@ -63,4 +64,36 @@ describe('Code Snippet', () => {
       expect(onClick).toBeCalled();
     });
   });
+
+  describe('check for showMoreBtn', () => {
+    let wrapper;
+
+    beforeEach(() => {
+      wrapper = mount(<CodeSnippet type={'multi'} />);
+    });
+
+    it('when less then 15 rows', () => {
+      wrapper.setProps({
+        children: '1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14',
+      });
+
+      expect(wrapper.find(Button).length).toBe(0);
+    });
+
+    it('when exactly 15 rows', () => {
+      wrapper.setProps({
+        children: '1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15',
+      });
+
+      expect(wrapper.find(Button).length).toBe(0);
+    });
+
+    it.skip('when more then 15 rows', () => {
```

Comment:
```suggestion
    it('when more then 15 rows', () => {
```

Reply:
I would love to have this unit test un-skipped.
However, when I do so it fails, and I am not sure why.

If I log the snapshot expect(wrapper).toMatchSnapshot() - the snapshot has 16 rows of data but the button is not there.
I can force the button render in CodeSnippet.js and the test will work - line 182: `{(true || shouldShowMoreLessBtn) && (`.
This indicates to me the setShouldShowMoreLessBtn react hook in onResize is not firing (debugged proved this).
This is why I wrote the testcase this way (setProps(children)) - I thought maybe I needed to invoke a resize.

So, I'm at a loss how to trigger the setShouldShowMoreLessBtn react hook to get the button to render.
Also confused how I can get 16 rows of data rendered and no button.